source/jormungandr/tests/direct_path_asgard_integration_tests.py

```diff
@@ -140,6 +160,7 @@ def test_journey_with_bike_direct_path(self):
         assert response['journeys'][0]['durations']['car'] == 500
         assert response['journeys'][0]['durations']['total'] == 500
         assert response['journeys'][0]['distances']['car'] == 50
+        assert response['journeys'][0]['sections'][0]['cycle_path_percentage'] == 0.0
```

Comment:
Why it is present inside a car section ? The field is not optional ?

Reply:
Fixed that! Thank you :)